### DISCUSSION TITLE
Remove old migrations after forced upgraded

### DISCUSSION THREAD

**Discussion by joostjager** [lightningnetwork/lnd#7087] [Ideas]
Now that there has been a forced upgrade to at least lnd 0.14, can some of the 30 migrations be deleted?

**Comment by guggero** [comment]
I think that would be a good moment to get rid of some old code, yes. I don't know of any reason why we couldn't do it with something like 0.16.

**Reply by yyforyongyu** [reply]
I'd love to see those removed!

**Reply by joostjager** [reply]
I suppose everything pre-0.14 can go now?
